### ISSUE TITLE
[Training loss] What's the training loss about the training phase?

### ISSUE THREAD

**Issue by jackieyung** [WeChatCV/Stand-In#57]
Thanks for your excellent work.

I'm trying to implement the training phase by using my own dataset, but I didn't find the training loss in the paper ( https://arxiv.org/pdf/2508.07901 ). Could you tell me the training loss while training?  @KBRASK 

Thanks a lot.

**Comment by KBRASK** [comment]
Hi @jackieyung ,

Thank you so much for your kind words and your interest in our work! I sincerely apologize for the late reply.

To answer your question, we didn't use any special or custom loss function during the training phase. Our approach is quite flexible and is designed to simply align with the standard training objective of the video base model.

Specifically, since we are using Wan 2.1 here, we just used its default flow matching loss.

Hope this helps with your implementation! Let me know if you have any other questions.
